php/editContent.php

```diff
@@ -1 +1,32 @@
 <?php
+
+require_once 'aboutMe.php';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>Abdi's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="editContent.css">
+</head>
+<body>
+<div class="content">
+    <h1>Edit About Me</h1>
+    <form action="update.php" method="post" class="formContent">
+        Edit Title:<input type="text" name="title" class="titleContent" value="<?php echo $title; ?>">
+        <br>
```

Comment:
<br> tags have pulled up with me in a code review before so might want to lose them before final story code review.
lines 19,20,23-25

Reply:
done but kept one